docs/docs/developers/reference/environment_reference/cli_reference.md

```diff
@@ -60,165 +433,463 @@ Options:
 ## Account Management
 
```

Comment:
```suggestion
Consider using the [`aztec-wallet`](./cli_wallet_reference.md) for account management (or contract interaction) related actions, since it has a PXE internally and manages aliases to get you started quicker.

`aztec` cli requires you to have a PXE running already (either as part of when you run the sandbox by default or just a separate PXE)
```